GH_UnitNumber/Components/SendToPostHog.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using Grasshopper.Kernel;
+using Newtonsoft.Json;
+using OasysGH;
+using OasysGH.Components;
+using OasysGH.Helpers;
+
+namespace GH_UnitNumber.Components {
+  public class SendToPostHog : GH_OasysComponent {
+    public override Guid ComponentGuid => new Guid("9da7ba33-62fc-4f36-81a3-7e24a09a644b");
+    public override GH_Exposure Exposure => GH_Exposure.hidden;
+    public override OasysPluginInfo PluginInfo => GH_UnitNumberPluginInfo.Instance;
+    protected override Bitmap Icon => null;
```

Comment:
no icon?

Reply:
This is a hidden component and since our icon designer has gone, no icon yes..